centaur/src/main/scala/centaur/serialization/TestsReports.scala

```diff
@@ -0,0 +1,46 @@
+package centaur.serialization
+
+import java.io.{FileInputStream, FileOutputStream}
+
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.twitter.chill.ScalaKryoInstantiator
+
+import scala.collection.mutable
+
+/**
+  * This class contains information about last version in which corresponding integration test was successfully run
+  * @param testCasesRuns  key is a test name; value is a version of Cromwell, in which this test passed last time
```

Comment:
Shouldn't we mark `value` as a parameter as well?

Reply:
As a value I meant not a separate parameter, but a part of Map's entry, <key, value>. Should we mark both key and value then?

Reply:
Actually, you're right. Let's leave it as it is